parastell/cubit_io.py

```diff
@@ -97,6 +97,10 @@ def export_mesh_cubit(filename, export_dir=""):
     subprocess.run(f"mbconvert {exo_path} {h5m_path}", shell=True)
     Path.unlink(exo_path)
 
+    # Delete any meshes present to prevent inclusion in future Cubit mesh
+    # exports
+    cubit.cmd(f"delete mesh volume all propagate")
```

Comment:
Do we always want to do this? Or should it be an option for the function?

Reply:
Supporting an option is probably a better idea